### DISCUSSION TITLE
SONAR.AM.C!g6

### DISCUSSION THREAD

**Discussion by nybbles2** [ramensoftware/windhawk#253] [General]
FYI, Norton is reporting the following all of a sudden:

Filename: windhawk.exe
Threat name: SONAR.AM.C!g6 
Full Path: c:\program files\windhawk\windhawk.exe

On computers as of 
4/27/2024 at 5:04:40 PM

Last Used 
8/21/2024 at 9:31:16 PM

Startup Item 
Yes
Launched 
Yes
Behavioral Protection monitors for suspicious program activity on your computer.

Few Users
Hundreds of users in the Norton Community have used this file.

New
This file was released 23 days  ago.

High
This file risk is high.

**Comment by m417z** [comment]
Answered here: https://github.com/ramensoftware/windhawk/issues/254#issuecomment-2307169272.
